modules/tinymce/changelog.txt

```diff
@@ -12,6 +12,7 @@ Version 5.4.0 (TBD)
     Fixed handling of mixed-case icon names by having the silver theme downcase them before icon retrieval #TINY-3854
     Fixed leading and trailing spaces lost when using `editor.selection.getContent({ format: 'text' })` #TINY-5986
     Fixed an issue where removing formatting in a table cell would cause Internet Explorer 11 to scroll to the end of the table #TINY-6049
+    Fixed an issue where editor.allow_html_data_urls was not being correctly applied #TINY-5951
```

Comment:
`editor.allow_html_data_urls` isn't clear, as that's not where people would use it. Setting names should be in backticks and `being` is normally a word that's avoided. So maybe this instead?
```
Fixed an issue where the `allow_html_data_urls` setting was not correctly applied #TINY-5951
```